postgres-derive/src/accepts.rs

```diff
@@ -40,6 +40,10 @@ pub fn enum_body(name: &str, variants: &[Variant]) -> TokenStream {
             return false;
         }
 
+        if #allow_mismatch {
+            return true;
+        }
+
```

Comment:
I'd rather just not generate the match below at all if allow_mismatch is set.

Reply:
true, fixed!